CesiumGltf/include/CesiumGltf/PropertyAttributePropertyView.h

```diff
@@ -122,34 +122,68 @@ class PropertyAttributePropertyView<ElementType, false>
    * @brief Constructs an invalid instance for a non-existent property.
    */
   PropertyAttributePropertyView() noexcept
-      : PropertyView<ElementType, false>(), _accessor{} {}
+      : PropertyView<ElementType, false>(), _accessor{}, _size{0} {}
 
   /**
    * @brief Constructs an invalid instance for an erroneous property.
    *
    * @param status The code from {@link PropertyAttributePropertyViewStatus} indicating the error with the property.
    */
   PropertyAttributePropertyView(PropertyViewStatusType status) noexcept
-      : PropertyView<ElementType, false>(status), _accessor{} {
+      : PropertyView<ElementType, false>(status), _accessor{}, _size{0} {
     assert(
         this->_status != PropertyAttributePropertyViewStatus::Valid &&
         "An empty property view should not be constructed with a valid status");
   }
 
+  /**
+   * @brief Constructs an instance of an empty property that specifies a default
+   * value. Although this property has no data, it can return the default value
+   * when {@link PropertyAttributePropertyView::get} is called. However,
+   * {@link PropertyAttributePropertyView::getRaw} cannot be used.
+   *
+   * @param classProperty The {@link ClassProperty} this property conforms to.
+   * @param size The number of elements in the primitive's POSITION accessor.
+   * Used as a substitute since no actual accessor is defined.
+   */
+  PropertyAttributePropertyView(
+      const ClassProperty& classProperty,
+      int64_t size) noexcept
+      : PropertyView<ElementType, false>(classProperty), _accessor{}, _size{0} {
+    // Don't override the status / size if something is wrong with the class
+    // property's definition.
+    if (this->_status != PropertyAttributePropertyViewStatus::Valid) {
+      return;
+    }
+
+    assert(
+        classProperty.defaultProperty &&
+        "Cannot construct a valid property view for an empty property with no "
+        "default value.");
```

Comment:
Given that this is part of the class's public API, would it be a bit safer to set the status to `ErrorNonexistentProperty` when there's no default? As written, it will assert in debug, which is fair enough because it's probably a developer error. But in a release build, it will pretend like there is a default, which doesn't seem great.